src/main/java/com/google/javascript/clutz/PlatformSymbols.java

```diff
@@ -0,0 +1,1716 @@
+          "ScriptEngineBuildVersion",
+          "ScriptEngineMajorVersion",
+          "ScriptEngineMinorVersion",
+          "ScriptProfile",
+          "ScriptProfileNode",
+          "ServiceWorkerClient",
+          "ServiceWorkerClientQueryOptions",
+          "ServiceWorkerClients",
+          "SharedWorker",
+          "SharedWorkerGlobalScope",
+          "SourceInfo",
+          "SpatialListener",
+          "SpatialPannerNode",
+          "StorageInfo",
+          "StorageQuota",
+          "TaskAttributionTiming",
+          "TextDecoder",
+          "TextEncoder",
+          "TextRange",
+          "Thenable",
+          "TouchInitDict",
+          "Transferable",
+          "TransformStream",
+          "TraversalDocument",
+          "TypeInfo",
+          "TypedArray",
+          "UserDataHandler",
+          "UserMessageHandler",
+          "UserMessageHandlersNamespace",
+          "VTTCue",
+          "VTTRegion",
+          "VibratePattern",
+          "ViewCSS",
+          "WEBGL_debug_shaders",
+          "WEBGL_draw_buffers",
+          "WEBGL_lose_context",
+          "WaveTable",
+          "WebGLVertexArrayObjectOES",
+          "WebKitAnimationEvent",
+          "WebKitBlobBuilder",
+          "WebKitNamespace",
+          "WebWorker",
+          "WorkerPerformance",
+          "XDomainRequest",
+          "XMLDOMDocument",
+          "XPathException",
+          "XPathNamespace",
+          "args",
+          "arguments",
+          "arrayLike",
+          "cancelIdleCallback",
+          "cancelRequestAnimationFrame",
+          "controlRange",
+          "createObjectURL",
+          "dump",
+          "escape",
+          "hasOwnProperty",
+          "mozCancelAnimationFrame",
+          "mozCancelRequestAnimationFrame",
+          "mozRequestAnimationFrame",
+          "msCancelAnimationFrame",
+          "msCancelRequestAnimationFrame",
+          "msRequestAnimationFrame",
+          "nsIDOMPageTransitionEvent",
+          "oCancelAnimationFrame",
+          "oCancelRequestAnimationFrame",
+          "oRequestAnimationFrame",
+          "openDatabase",
+          "opera",
+          "requestFileSystem",
+          "requestIdleCallback",
+          "resolveLocalFileSystemURI",
+          "revokeObjectURL",
+          "symbol",
+          "thenable",
+          "undefined",
+          "unescape",
+          "webCrypto",
+          "webkit",
+          "webkitAudioContext",
+          "webkitAudioPannerNode",
+          "webkitCancelRequestAnimationFrame",
+          "webkitIDBCursor",
+          "webkitIDBKeyRange",
+          "webkitIDBRequest",
+          "webkitIDBTransaction",
+          "webkitIDBVersionChangeEvent",
+          "webkitMediaStream",
+          "webkitOfflineAudioContext",
+          "webkitPannerNode",
+          "webkitRequestFileSystem",
+          "webkitResolveLocalFileSystemURI",
+          "webkitURL");
+
+  /** Set of all symbols exposed by TypeScript's lib.*.d.ts. */
```

Comment:
can you at least throw the little ts extractor in a gist here

Reply:
done